static/js/navbar_alerts.js

```diff
@@ -104,21 +112,35 @@ export function show_profile_incomplete(is_profile_incomplete) {
 export function initialize() {
     const ls = localstorage();
     if (page_params.insecure_desktop_app) {
-        open($("[data-process='insecure-desktop-app']"));
+        open(
+            {data_process: "insecure-desktop-app", custom_class: "red"},
+            render_insecure_desktop_app_alert_content,
+        );
     } else if (page_params.server_needs_upgrade) {
         if (should_show_server_upgrade_notification(ls)) {
-            open($("[data-process='server-needs-upgrade']"));
+            open(
+                {data_process: "server-needs-upgrade", custom_class: "red"},
+                render_server_needs_upgrade_alert_content,
+            );
         }
     } else if (page_params.warn_no_email === true && page_params.is_admin) {
         // if email has not been set up and the user is the admin,
         // display a warning to tell them to set up an email server.
-        open($("[data-process='email-server']"));
+        open(
+            {data_process: "email-server", custom_class: "red"},
+            render_configure_email_alert_content,
+        );
     } else if (should_show_notifications(ls)) {
-        open($("[data-process='notifications']"));
+        open({data_process: "notifications"}, render_notifications_alert_content);
     } else if (unread_ui.should_display_bankruptcy_banner()) {
-        open($("[data-process='bankruptcy']"));
+        const unread_msgs_count = page_params.unread_msgs.count;
+        open(
+            {data_process: "bankruptcy", custom_class: "brankruptcy", is_bankruptcy_alert: true},
```

Comment:
There seems to be a typo here.

Reply:
Should we remove this class (`bankruptcy`)? As we are not using it in CSS or JS?